src/wallet/rpcwallet.cpp

```diff
@@ -1097,6 +1097,137 @@ UniValue easyreceive(const JSONRPCRequest& request)
             coin_control);
 }
 
+UniValue easysendinvite(const JSONRPCRequest& request)
+{
+
+    CWallet * const pwallet = GetWalletForJSONRPCRequest(request);
+    if (!EnsureWalletIsAvailable(pwallet, request.fHelp)) {
+        return NullUniValue;
+    }
+
+    if (request.fHelp || request.params.size() > 3)
+        throw std::runtime_error(
+            "easysend (amount \"password\", blocktimeout)\n"
+            "\nSend invites to a given channel.\n"
+            + HelpRequiringPassphrase(pwallet) +
+            "\nArguments:\n"
+            "1. \"amount\"             (numeric or string) Optional amount of invites to send. eg 5\n"
+            "2. \"password\"           (string) Optional password to further secure the transaction.\n"
+            "3. blocktimeout           (numeric) The amount of blocks the transaction can be buried until the receiver cannot accept funds\n"
+            "\nResult:\n"
+            "\"txid\"                  (string) The transaction id.\n"
+            "\"pub\"                   (string) Escrow public key in hex.\n"
+            "\nExamples:\n"
+            + HelpExampleCli("easysendinvite", "")
+            + HelpExampleCli("easysendinvite", "1 abc124 100")
+        );
+
+    ObserveSafeMode();
+    LOCK2(cs_main, pwallet->cs_wallet);
+
+
+    // Amount
+    int amount = 1;
+    if (!request.params[0].isNull()) {
+        amount = request.params[0].get_int();
+        if (amount < 0) {
+            throw JSONRPCError(RPC_INVALID_PARAMS, "Invalid amount for send");
+        }
+    }
+
+    std::string optional_password = "";
+    if (!request.params[1].isNull()) {
+        optional_password = request.params[1].get_str();
+    }
+
+    int max_blocks = 10080; // about a week.
+    if(!request.params[2].isNull()) {
+        max_blocks = request.params[2].get_int();
+    }
+
+    if (max_blocks < 1) {
+        throw JSONRPCError(RPC_INVALID_PARAMS, "maxblocks must be greater than 0");
+    }
+
+    EnsureWalletIsUnlocked(pwallet);
+
+    CAmount balance = pwallet->GetBalance(true);
```

Comment:
I forget, does this balance include the confirmation invite?

Reply:
no, it's subtracted from the result for each address